ci/build_container/build_container_ubuntu.sh

```diff
@@ -25,10 +41,18 @@ update-alternatives --config g++
 update-alternatives --config gcov
 # Bazel and related dependencies.
 apt-get install -y openjdk-8-jdk curl
-echo "deb [arch=amd64] https://storage.googleapis.com/bazel-apt stable jdk1.8" | tee /etc/apt/sources.list.d/bazel.list
-curl https://bazel.build/bazel-release.pub.gpg | apt-key add -
-apt-get update
-apt-get install -y bazel
+case $ARCH in
+    'ppc64le' )
+        curl -fSL https://oplab9.parqtec.unicamp.br/pub/ppc64el/bazel/ubuntu_16.04/bazel_bin_ppc64le_0.25.2 -o /usr/local/bin/bazel
```

Comment:
Hmm, can we make this always latest released? You can do a fixed URL here, or do something similar to the CentOS script.